Portfolio/src/App.vue

```diff
@@ -3,96 +3,95 @@ import { ref } from 'vue';
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; " to="/work">Work</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/about">About</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/contact">Contact</RouterLink>
-            </li>
-        </ul>
-
-        <div class="menu-fullName">
-            <h1 class="menu-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="menu-lastName">{{ homeViewContent.lastName }}</h1>
+    <Wrapper>
+        <header>
+            <div class="router-wrapper">
+                <button @click="isMenuShown = !isMenuShown" class="menuButton">
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }">
+                    </div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                </button>
+
+                <nav>
+                    <ul class="navbar">
+                        <li>
+                            <RouterLink to="/">Home
+                            </RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to="/work">Work</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to=" /about">About</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to="/contact">Contact
+                            </RouterLink>
+                        </li>
+                    </ul>
+                </nav>
+
+            </div>
+        </header>
+
+        <div v-if="isMenuShown" class="openedMenu" style=" text-transform: uppercase;">
+            <ul class="menu-links">
+                <li>
+                    <RouterLink to="/">Home
+                    </RouterLink>
+                </li>
+                <li>
+                    <RouterLink to="/work">Work</RouterLink>
+                </li>
+                <li>
+                    <RouterLink to=" /about">About</RouterLink>
+                </li>
+                <li>
+                    <RouterLink to="/contact">Contact</RouterLink>
+                </li>
+            </ul>
+
+            <div class="menu-fullname">
+                <h1 class="menu-firstName">{{ homeViewContent.firstName }}</h1>
+                <h1 class="menu-astName">{{ homeViewContent.lastName }}</h1>
+            </div>
         </div>
 
+        <!-- <footer>
+        <Copyright />
+    </footer> -->
 
-
-    </div>
-
-    <RouterView />
+        <RouterView />
+    </Wrapper>
 </template>
 
 <style>
 html,
 body {
     background-color: #D4ABB2;
     font-family: "Raleway";
-    text-transform: uppercase;
 }
 
 .menuButton {
```

Comment:
The classic convention that we use at Telia, is that we use kebab-case instead of camelCase for classnames. So for example instead of `menuButton `we write `menu-button`. Similar to the rest of the classnames

https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/

Reply:
Done